examples/apeworx_everlasting_bots.py

```diff
@@ -410,15 +411,31 @@ def do_trade(trade):
     """Execute agent trades on hyperdrive solidity contract"""
     agent_key = f"agent_{trade.wallet.address}"
     trade_amount = to_fixed_point(trade.trade_amount)
-    # if trade.action_type.name in ["ADD_LIQUIDITY", "REMOVE_LIQUIDITY"]:
-    #    continue  # todo
-    if trade.action_type.name == "OPEN_SHORT":
+    if trade.action_type.name == "ADD_LIQUIDITY":
```

Comment:
this is the only functional change, everything else is stylistic (sourcery), type-checks (pyright) or pylint